src/Stratis.Bitcoin/Utilities/AsyncLoop.cs

```diff
@@ -115,11 +115,15 @@ private Task StartAsync(CancellationToken cancellation, TimeSpan? delayStart = n
             return Task.Run(async () =>
             {
                 Exception uncaughtException = null;
-                this.logger.LogInformation(this.Name + " starting.");
                 try
                 {
                     if (delayStart != null)
+                    {
+                        this.logger.LogInformation($"{this.Name} starting in {delayStart.Value.TotalSeconds} seconds.");
```

Comment:
don't use string interpolation in logs

Reply:
fixed.
btw there are lot of logs actually using string interpolation, like this
https://github.com/stratisproject/StratisBitcoinFullNode/blob/a0004c768207fcaf624bdc88ddf40ca3316000f2/src/Stratis.Bitcoin.Features.MemoryPool/MempoolValidator.cs#L1027

that should be fixed too, what's the best thing to do, change it when you put hands on a file containing wrong logs or fix with a PR whenever spot some?

P.S.
for the same reason, using previous `this.Name + "some test" should be discouraged too

Reply:
>btw there are lot of logs actually using string interpolation, like this

Yeah, this is incorrect

>change it when you put hands on a file containing wrong logs or fix with a PR whenever spot some?

Fix it whenever you spot it around the code you are working on. Or just create a separated PR for this and nothing else.